spec/jobs/external_contibutions_processor_job_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+require 'sidekiq/testing'
+
+RSpec.describe ExternalContributionsProcessorJob do
+  it 'executes external contributions processor' do
+    expect(Processors::External::Contributions).to receive(:call)
```

Comment:
I'm ok with this test, but I see it as an integration test, so to add more value I'd have checked the happy path, that something was created or whatever